spec/javascripts/ContributorsReadMoreSpec.js

```diff
@@ -0,0 +1,17 @@
+describe("ContributorsReadMore", function() {
```

Comment:
Good tests! I think you should also test the click event on `$('.read-more-show')` too.